File: .Net/RecognizersTextPerformanceTest/Services/ConfigsReader.cs
using RecognizersTextPerformanceTest.Helpers;
using RecognizersTextPerformanceTest.ViewModels;
using System;
using System.IO;

namespace RecognizersTextPerformanceTest.Services
{
    public static class ConfigsReader
    {
        public static ConfigModel LoadApplicationConfigs()
        {
            var filePath = Path.Combine(Constants.ConfigsFileDirectory, Constants.ConfigsFileName);
            if (File.Exists(filePath))
            {
                var configsFile = File.ReadAllText(filePath);
                return JsonHandler.DeserializeObject<ConfigModel>(configsFile, Constants.ConfigsFileName);
            }
            // TODO: handle exception
            throw new Exception("File not found");
        }
    }
}

File: Python/services/configs_reader.py
import os
import constants
import json

def load_application_configs():
    configs_file_path = os.path.join(constants.configs_file_directory, constants.configs_file_name)

    if os.path.exists(configs_file_path):
        with open(configs_file_path) as f:
            configs = f.read() 
        return json.loads(configs)

    raise Exception("configs file missing")